include/gz/rendering/ParticleEmitter.hh

```diff
@@ -47,8 +47,11 @@ namespace gz
       /// \brief Ellipsoid emitter.
       EM_ELLIPSOID        = 3,
 
+      /// \brief Cone emitter.
+      EM_CONE             = 4,
```

Comment:
ogre 1.x and ogre 2.x do not support cone [particle emitters](https://ogrecave.github.io/ogre/api/1.12/_particle-_scripts.html#Standard-Particle-Emitters). So let's not add it to the enums yet until we have an engine that supports it.